src/main.rs

```diff
@@ -43,15 +44,15 @@ fn main() {
         .get_matches_from(std::env::args());
 
     if matches.get_flag("backup") {
-        eprintln!("The --backup flag is no longer supported. Please use --out or --dir to preserve your existing files.");
-        exit(1)
+        error!("The --backup flag is no longer supported. Please use --out or --dir to preserve your existing files.");
```

Comment:
I realise I'm a little late, but this needs to be `eprintln` as we haven't initialised the env_logger yet. (And it was intended to not be silenceable with the `-q` flag)
That said, it may be time to remove this completely now.

Reply:
PR #663 was opened for this.